srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml

```diff
@@ -436,6 +436,15 @@ EKTField = FullEKTField / ShortEKTField / ExtensionEKTField
               the previous step are saved in a map indexed by the SSRC
               found in the EKTPlaintext and can be used for any future
               crypto operations on the inbound packets with that SSRC.
+              If the SRTP Master Key recovered from the EKTPlaintext
+              is longer than needed by SRTP transfrom in use, the
+              first bytes are used. If the SRTP Master Key recovered
+              from the EKTPlaintext is shorter than needed by SRTP
+              transfrom in use, then the bytes received replace the
+              first bytes in the existing key but the other bytes
+              after that remain the same as the old key. This allows
```

Comment:
Do you want to explicitly allow to only replace for example the first byte, or should there be a definition of a minimum length to execute such a partial replace?